Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs

```diff
@@ -65,30 +82,82 @@ public void AddTip(SceneTips.Tip tip)
 
+            view.Image.sprite = icon;
+
+            LoadingBackgroundView background = Instantiate(backgroundImageTemplate, backgroundParent);
+            background.Image.sprite = icon;
+
             TipBreadcrumb breadcrumb = Instantiate(breadcrumbPrefab, breadcrumbParent);
             int breadcrumbIndex = tipsBreadcrumbs.Count;
             breadcrumb.Button.onClick.AddListener(() => OnBreadcrumbClicked?.Invoke(breadcrumbIndex));
 
             tips.Add(view);
             tipsBreadcrumbs.Add(breadcrumb);
+            backgrounds.Add(background);
         }
 
         public void ShowTip(int index)
         {
-            foreach (TipView? view in tips)
-                view.gameObject.SetActive(false);
-
             tips[index].gameObject.SetActive(true);
-            backgroundImage.sprite = tips[index].Image.sprite;
+            backgrounds[index].gameObject.SetActive(true);
 
             foreach (TipBreadcrumb? breadcrumb in tipsBreadcrumbs)
                 breadcrumb.Unselect();
 
             tipsBreadcrumbs[index].Select();
         }
+
+        public void HideAllTips()
+        {
+            foreach (TipView? view in tips)
+                view.gameObject.SetActive(false);
+
+            foreach (LoadingBackgroundView background in backgrounds)
+                background.gameObject.SetActive(false);
+        }
+
+        public async UniTask ShowTipWithFadeAsync(int index, float duration, CancellationToken ct)
+        {
+            ShowTip(index);
+
+            tips[index].RootCanvasGroup.alpha = 0f;
+            backgrounds[index].RootCanvasGroup.alpha = 0f;
+
+            UniTask tipFadeTask = tips[index]
+                                 .RootCanvasGroup.DOFade(1f, duration)
+                                 .AsyncWaitForCompletion()
+                                 .AsUniTask();
+
+            UniTask backgroundFadeTask = backgrounds[index]
+                                        .RootCanvasGroup.DOFade(1f, duration)
+                                        .AsyncWaitForCompletion()
+                                        .AsUniTask()
+                                        .AttachExternalCancellation(ct);
+
+            await UniTask.WhenAll(tipFadeTask, backgroundFadeTask);
+        }
+
+        public async UniTask HideTipWithFadeAsync(int index, float duration, CancellationToken ct)
+        {
+            UniTask tipFadeTask = tips[index]
+                                 .RootCanvasGroup.DOFade(0f, duration)
+                                 .AsyncWaitForCompletion()
+                                 .AsUniTask();
+
+            UniTask backgroundFadeTask = backgrounds[index]
+                                        .RootCanvasGroup.DOFade(0f, duration)
+                                        .AsyncWaitForCompletion()
+                                        .AsUniTask()
+                                        .AttachExternalCancellation(ct);
```

Comment:
These extensions also have a built-in `CancellationToken` support so you don't need to work it around with `AttachExternalCancellation`. 
`AttachExternalCancellation` is general is a bad practive and should be used for code out of our control only: it does not clean-up any resources under the hood: it stops spininng the wrapped task only that may lead to resources leak.